valkey.conf

```diff
@@ -2374,17 +2373,24 @@ rdb-save-incremental-fsync yes
 # active-defrag-threshold-upper 100
 
 # Minimal effort for defrag in CPU percentage, to be used when the lower
-# threshold is reached
+# threshold is reached.
+# Note: this is not actually a cycle time, but is an overall CPU percentage
 # active-defrag-cycle-min 1
```

Comment:
The first sentence already says it's in CPU percentage. No need to mention the same thing twice. Edit the first sentence to make it clearer instead, if needed.

Reply:
If we want to keep the old names "cycle-min" and "cycle-max", I think the note is a necessary clarification.

Reply:
I don't know how to reconcile these two comments. Do people like something like this?
```
# Minimal effort for defrag in CPU percentage, not cycle time as the name might
# suggest, to be used when the lower threshold is reached.
```

Reply:
Yes! LGTM